docs/capabilities.md

```diff
@@ -4,33 +4,103 @@ time.  Whether you are working with large legacy codebases, your own
-    a unified manner to appropriate team members to greatly reduce
-    context-switching costs and incident time-to-resolution.
--   Managing a service life cycle, from commits to CI builds to
-    deployments to alerts to corrective action
--   Automating common tasks in software development and operation,
-    for example:
-    -   Using Rug editors to modify code in a consistent mannger
-        across many repositories
-    -   Ensuring a project conforms with your standards using a Rug
-        reviewer
-
-# What you can do with Atomist
-## Rug is the eventing runtime that ties everything together
+## Bringing everything together
+
+Atomist consumes events from the platforms and tools you use every day.
+By linking related events from source code repositories, continuous
+integration, testing, and runtime environments, Atomist can present
+them in a unified manner to appropriate team members via direct or
+group messages.  The result is massive reductions in context-switching
+costs and and shorter incident time-to-resolution.
+
+<!-- *That rug really tied the room together, did it not?* -->
+
+Rug is the eventing runtime that ties everything together.  Rug not
+only understands individual events, but can learn relationships
+between events and respond to events with appropriate, automated
+actions.
+
 ## Development lifecycle automation
-### Event-driven actions: handlers
-### ChatDev and ChatOps: commands
-### Project updates: editors
-### Project compliance: reviewers
-### Project creation: generators
-## Events and code selection: tree expressions
-## Interfaces
-### XX Button XX
-### Bot
-### CLI
+
+Atomist helps you develop better software more quickly by greatly
+enhancing your ability to automate your software development
+lifecycle.  By intelligently linking your source code repository, your
+CI system, your deployment platform, your logs, and your alerting
+system, Atomist connects code changes to runtime changes and provides
+that information back, with its full context, back to the appropriate
+team member.  It can can even automate remediation, obviating the need
+to alert anyone.
+
+The Rug programming model provides several different types of Rugs to
+help you manage your development and operations lifecycle.
+
+<!-- can we improve the layout of the items below? -->
+
+**Event-driven actions: <span class="rugs">[Handlers][handlers]</span>** - Rug
+handlers respond to events.  Handlers can use the information on an
+event and its related events to message the right team members, take
+direct action, and/or create new events, which can trigger other
+handlers.
+
+**ChatDev and ChatOps: <span class="rugs">[Commands][commands]</span>** - Rug
+commands are tasks you can execute within chat, automating common
+tasks in software development and operation.  Want to release a new
+version of your service to production?  Do it right in chat with a
+command.
+
+**Project updates: <span class="rugs">[Editors][editors]</span>** -
```

Comment:
I think we need to say "whether you created the project with an Atomist generator or not". that isn't actually obvious to most.